auth.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Required External Modules
+ */
+
+const express = require('express');
+const router = express.Router();
+const passport = require('passport');
+const querystring = require('querystring');
+
+require("dotenv").config();
+
+/** 
+ * Routes Definitions
+*/
+
+router.get(
+  '/login',
+  passport.authenticate('auth0', {
+    scope: "openid email profile"
+  }),
+  (req, res) => {
+    console.log(req.query);
+    res.send({ hello: 'hi' });
+    res.end();
+  }
+);
+
+router.get('callback', (req, res, next) => {
+  passport.authenticate('auth0', (err, user, info) => {
+    if (err) {
+      return next(err);
+    }
+
+    if (!user) {
+      return res.redirect('/login');
+    }
+
+    req.logIn(user, (error) => {
+      if (error) {
+        return next(error);
+      }
+      const returnTo = req.session.returnTo;
```

Comment:
this can be destructured

```suggestion
      const {returnTo} = req.session;
```